# Página 1

Divergent Effects of PERK and IRE1 Signaling on Cell
Viability
Jonathan H. Lin1,2*, Han Li1,2, Yuhong Zhang3, David Ron3, Peter Walter1,2
1 Howard Hughes Medical Institute, University of California San Francisco, San Francisco, California, United States of America, 2 Department of Biochemistry and
Biophysics, University of California San Francisco, San Francisco, California, United States of America, 3 Departments of Cell Biology and Medicine, Kimmel Center for
Biology and Medicine, Skirball Institute, New York University School of Medicine, New York, New York, United States of America
Abstract
Protein misfolding in the endoplasmic reticulum (ER) activates a set of intracellular signaling pathways, collectively termed
the Unfolded Protein Response (UPR). UPR signaling promotes cell survival by reducing misfolded protein levels. If
homeostasis cannot be restored, UPR signaling promotes cell death. The molecular basis for the switch between prosurvival
and proapoptotic UPR function is poorly understood. The ER-resident proteins, PERK and IRE1, control two key UPR
signaling pathways. Protein misfolding concomitantly activates PERK and IRE1 and has clouded insight into their
contributions toward life or death cell fates. Here, we employed chemical-genetic strategies to activate individually PERK or
IRE1 uncoupled from protein misfolding. We found that sustained PERK signaling impaired cell proliferation and promoted
apoptosis. By contrast, equivalent durations of IRE1 signaling enhanced cell proliferation without promoting cell death.
These results demonstrate that extended PERK and IRE1 signaling have opposite effects on cell viability. Differential
activation of PERK and IRE1 may determine life or death decisions after ER protein misfolding.
Citation: Lin JH, Li H, Zhang Y, Ron D, Walter P (2009) Divergent Effects of PERK and IRE1 Signaling on Cell Viability. PLoS ONE 4(1): e4170. doi:10.1371/
journal.pone.0004170
Editor: Andreas Bergmann, UT MD Anderson Cancer Center, United States of America
Received October 16, 2008; Accepted December 6, 2008; Published January 12, 2009
Copyright:  2009 Lin et al. This is an open-access article distributed under the terms of the Creative Commons Attribution License, which permits unrestricted
use, distribution, and reproduction in any medium, provided the original author and source are credited.
Funding: This work was supported by NIH grants DK047119, ES08681, EY018313, and GM032384, and the Cystic Fibrosis Foundation. Peter Walter is an
Investigator of the Howard Hughes Medical Institute. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of
the manuscript.
Competing Interests: The authors have declared that no competing interests exist.
* E-mail: JLin@ucsd.edu
Introduction
Physiologic or pathologic processes that disturb protein folding in
the endoplasmic reticulum (ER) activate a set of signaling pathways
termed the Unfolded Protein Response (UPR). The molecular
gatekeepers of the UPR are ER-resident transmembrane proteins
that monitor the quality of protein folding in the ER and relay that
information to the rest of the cell. In mammalian cells, PERK and
IRE1 independently govern two key UPR signal transduction
pathways [1]. PERK is a transmembrane kinase that phosphorylates
translation initiation factor eIF2a, thereby reducing cellular protein
synthesis and with it the load of proteins entering into the ER [2].
eIF2a phosphorylation also allows the translation of select mRNAs
that contain small open reading frames in their 59 untranslated
regions, leading to the production of transcription activators, such as
ATF4 and ATF5 [3,4]. IRE1 is a bifunctional transmembrane
kinase/endoribonuclease that induces the non-conventional splicing
of Xbp1 mRNA to produce another b-ZIP transcription activator,
XBP1 [5]. In addition to splicing Xbp1 mRNA, IRE1’s kinase can
also activate the c-Jun N-terminal kinase (JNK) signaling pathway
through the MAP3K cascade [6,7]. The transcription factors
produced by PERK, IRE1, and other UPR signaling pathways
collaborate to control behavior, metabolism, and ultimately cell fate
in response to ER stress by inducing a wide array of targets that
include protein folding chaperones such as ERdj4 [8] and additional
transcriptional activators such as Chop [3].
Genetic and pharmacological experiments have demonstrated
that PERK signaling can confer both protective and proapoptotic
effects in the face of ER stress. For instance, genetic deletion of Perk
or impairment of eIF2a activity impaired cell survival [9,10].
Conversely, transient artificial PERK activation or pharmacolog-
ical eIF2a activation enhanced cell survival in response to ER
protein misfolding [11,12]. Deletion of downstream components
of PERK signaling, Atf4 and Chop, impaired or enhanced cell
survival in response to protein misfolding depending on the cell
type studied [3,13,14,15,16,17].
Like PERK, IRE1 signaling has also been implicated in
enhancing or impairing cell survival. Artificial extension of IRE1’s
RNAse function enhanced cell survival in the face of ER stress
[18,19]. RNAi knockdown of Xbp1, IRE1’s RNAse target,
impaired cell survival after protein misfolding in vitro and was
required for the survival of multiple secretory cell types in vivo
[20,21]. Genetic deletion of Ask1, the MAP3K proposed to link
IRE1 signaling to JNK, conferred resistance to ER stress-induced
cell death [7,22]. JNK can prevent or promote cell death
depending on the specific stimuli, intensity, and/or duration of
activation [23,24,25].
These findings demonstrate that PERK and IRE1 signaling can
regulate cell survival after protein misfolding. How do cells
modulate PERK and IRE1 activities to arrive at either cell fate?
Previous studies demonstrated that the duration of PERK and
IRE1 signaling varied markedly after the imposition of protein
misfolding [18]. In particular, chronic ER stress led to IRE1
branch inactivation while PERK signaling was unaffected. These
observations suggested that the progression toward cell death from
unmitigated protein misfolding involved attenuation of IRE1
PLoS ONE | www.plosone.org
1
January 2009 | Volume 4 | Issue 1 | e4170


# Página 2

signaling coupled with persistent PERK activity. Previously, we
employed chemical-genetic tools to artificially activate IRE1 and
demonstrated a cytoprotective effect for its RNAse function in
isogenic human cells [18]. Here, we employed a similar strategy to
selectively activate PERK. We observed that sustained PERK
signaling was detrimental to cell viability whereas the equivalent
duration of IRE1 signaling was not, suggesting that extended
PERK activity contributes to the cell death that occurs with
chronic ER stress.
Results
Chemical-Genetic Control of PERK and IRE1 Signaling in
Human Cells
We previously used recombinase-directed site-specific DNA
integration to introduce alleles into the genome of human
embryonic kidney 293 (HEK293) cells [18]. This technique
minimized perturbation of the native UPR as well as differences
arising from position insertion variegation effects. We extended
this strategy to create additional isogenic cell lines stably
expressing an artificial PERK allele, Fv2E-Perk, which had
previously
been
demonstrated
to
activate
wild-type
PERK
signaling in hippocampal neurons and CHO cells upon addition
of the dimerizing molecule, AP20187 [11,26]. We observed stable
Fv2E-Perk mRNA and protein expression at all times examined in
our cells (Fig. 1A and Fig. S1). To determine how effectively we
could recapitulate PERK branch signaling in HEK293 cells
expressing Fv2E-Perk, we monitored multiple specific parameters
of PERK activity after addition of the dimerizing agent, AP20187.
After application of drug, we observed production of phosphor-
ylated eIF2a and a downstream translational target ATF4 that
approached levels seen with exposure to thapsigargin, an ER toxin
that strongly induces all branches of the UPR (Fig. 1A and Fig.
S1). Consistent with activation of these proximal parameters of
PERK branch activity, we also observed increased mRNA levels of
downstream PERK signaling transcriptional targets, Chop and
Gadd34, after AP20187 application (Fig. 1A and Fig. S1). The
GADD34 phosphatase has been demonstrated to target phos-
phorylated eIF2a and thereby deactivate PERK branch signaling
[27]. Interestingly, we observed no diminution in phosphorylated
eIF2a levels in the presence of AP20187, even though Gadd34 was
induced, suggesting that drug-activated Fv2E-PERK overcame the
negative feedback effects of GADD34 on eIF2a (Fig. 1A). Lastly,
to determine if AP20187’s effects were confined to PERK or had
non-specifically triggered ER stress, we examined a specific
marker of IRE1 activation, splicing of Xbp1 mRNA. Cells
expressing Fv2E-Perk spliced Xbp-1 mRNA in response thapsi-
gargin, but no Xbp1 mRNA splicing was observed at all
concentrations and durations of AP20187 exposure that activated
Fv2E-PERK (Fig. 1B and Fig. S1). Hence, these cells provide a
system to examine the effects of selective PERK branch signaling.
To study the effects of selective IRE1 branch activity on cell
viability, we used transgenic HEK293 cells expressing an artificial
Ire1[I642G] allele which we had previously shown could be
regulated by addition of the ATP analogue, 1NM-PP1 [18,19]. As
a control for the specificity of IRE1[I642G]’s effects, we created
additional cells expressing an allele of IRE1, Ire1[I642G/K599A],
that bore a second missense mutation at amino acid position 599,
which converted an essential lysine residue to alanine in the
catalytic kinase domain of IRE1 [28]. We observed stable
expression of IRE1[I642G] or IRE1[I642G/K599A] protein at
all times examined in transgenic HEK293 cells (Fig. 2A).
However, 1NM-PP1 application triggered Xbp1 mRNA splicing
and ERdj4 mRNA induction, two parameters of IRE1 branch
signaling, only in cells expressing IRE1[I642G], indicating that the
additional
K599A
point
mutation
in
IRE1[I642G/K599A]
abolished its activity (Fig. 2A). To ascertain that 1NM-PP1’s
effects were confined to activation of IRE1[I642G], we examined
a marker of PERK branch signaling, production of ATF4 protein.
Parental and transgenic cells produced ATF4 in response to
thapsigargin treatment, but no ATF4 protein was observed at all
durations of 1NM-PP1 treatment that activated IRE1 signaling
(Fig. 2B). Hence, these cells provided a system to examine the
effects of selective IRE1 branch signaling on cell viability.
Divergent Effects of Extended PERK and IRE1 Signaling
on Cell Proliferation and Apoptosis
We used these isogenic cell lines bearing Fv2E-Perk, Ire1[I642G],
or Ire1[I642G/K599A] to address how selective IRE1 or PERK
signaling affected cell viability with respect to proliferation and
Figure 1. Selective and specific activation of PERK signaling. (A)
Parental wild-type and transgenic HEK293 cells expressing the AP20187-
sensitized Fv2E-Perk allele were treated for the indicated times with
AP20187 (2 nM). Fv2E-Perk, Gadd34, and Chop mRNA levels were
measured by quantitative PCR, normalized to levels of a housekeeping
gene, Rpl19, and are shown relative to levels in untreated cells. Fv2E-
PERK, phospho-eIF2a, and ATF4 proteins were detected by immuno-
blotting. Total eIF2a protein was measured as a loading control. (B)
Parental wild-type and transgenic HEK293 cells expressing the AP20187-
sensitized Fv2E-Perk allele were treated for the indicated times with
AP20187 (2 nM). Xbp1 mRNA splicing was assesed by RT-PCR. The
unspliced (u) and spliced (s) Xbp1 mRNA products are indicated as
labeled. The asterisk indicates the position of a hybrid amplicon.
doi:10.1371/journal.pone.0004170.g001
PERK, IRE1, and Apoptosis
PLoS ONE | www.plosone.org
2
January 2009 | Volume 4 | Issue 1 | e4170


# Página 3

apoptosis. Chronic protein misfolding induced by multi-day
exposure to tunicamycin or thapsigargin severely impaired cell
proliferation and triggered apoptosis in wild-type cells (Video S1,
Fig. 3A, Fig. 4). When we selectively activated PERK signaling in
cells bearing Fv2E-Perk by application of AP20187 for up to
48 hours, we observed a pronounced reduction in cell numbers
compared to mock treated or parental cells exposed to AP20187
(Video S2, Fig. 3A, Fig. 3B). By contrast, when we selectively
activated IRE1 signaling in transgenic cells bearing Ire1[I642G] by
application of 1NM-PP1, we observed increased cell numbers
compared to mock treated or parental cells exposed to the drug
(Video S3, Fig. 3A, Fig. 3C). The advantage in proliferation
specifically required functional IRE1 branch activity, since 1NM-
PP1 exposure did not enhance survival in cells bearing the doubly-
mutated Ire1[I642G/K599A] allele (Fig. 3A, Fig. 3B). In sum, these
studies clearly demonstrated that sustained PERK signaling impairs
cell proliferation while IRE1 signaling promotes cell growth.
We also observed striking morphologic changes in cells, in
which
PERK
signaling
was
selectively
activated,
including
retraction of cellular extensions, loss of refractiveness under
phase-contrast microscopy, and detachment from the underlying
matrix (Video S2, Fig. 3A). These physical changes resembled
those seen in cells undergoing cell death after exposure to lethal
concentrations of ER stress-inducing agents, such as tunicamycin
or thapsigargin (Video S1, Fig. 3A). By contrast, none of these
morphologic changes were seen in cells in which IRE1 branch
signaling had been selectively activated (Video S3, Fig. 3A). These
morphologic changes suggested that sustained PERK activity
triggered apoptosis in addition to impairing cell proliferation.
To determine if molecular markers of apoptosis occurred in
these cells, we next examined cleavage of poly(ADP-ribose)
polymerase (PARP), a nuclear DNA repair enzyme that undergoes
proteolysis in response to many apoptotic stimuli [29]. Robust
production of cleaved PARP was observed after 48 to 72 hours of
exposure to tunicamycin or thapsigargin (Fig. 4 and Fig. S2).
Minimal PARP cleavage was seen in wild-type cells exposed to
1NM-PP1 or AP20187, indicating that these small molecules did
not trigger cell death at bio-efficacious concentrations (Fig. S2).
When AP20187 was applied to cells expressing Fv2E-PERK, we
saw strong production of cleaved PARP (Fig. 4). By contrast, when
1NM-PP1 was added to cells expressing IRE1[I642G], PARP was
not cleaved (Fig. 4). Taken together with the cytomorphologic
changes, these findings indicate that sustained PERK signaling
triggers apoptosis, whereas IRE1 signaling does not when
activated for equivalent duration. Consistent with the incompat-
ibility of extended PERK signaling with viability, loss of the Fv2E-
PERK transgene was observed in all cells that were able to
proliferate in the presence of AP20187 (Fig. S3).
Discussion
The UPR detects and responds to ER protein misfolding
acutely by enhancing the protein folding capacity of the ER, but, if
protein misfolding persists, the UPR promotes cell death. The
molecular basis for this switch between protective and proapopto-
tic UPR function is poorly understood. Prior studies from our
group had delineated distinct molecular phases of UPR signaling
in which acute ER stress activated both PERK and IRE1, but
persistent chronic ER stress activated only the PERK pathway and
attenuated IRE1 signaling [18]. These observations led to the
hypothesis that the switch in IRE1 signaling coupled with
unabated PERK activity contributed to the transition from
protective to proapoptotic UPR function. To examine this model,
we used chemical-genetic approaches to activate PERK or IRE1
in isolation in isogenic ‘‘sister’’ human cell lines and observed that
chronic PERK signaling promotes cell death. By contrast, IRE1
activity enhances cell survival. Coupled with our previous studies,
these findings provide compelling evidence that the time course of
PERK and IRE1 signaling plays a critical role in determining how
the UPR selects between life and death cell fates.
Our current finding that chronic PERK activity impairs cell
viability is consistent with our prior study showing that selective
activation of PERK triggered cell death in other cell types [11] as
well numerous reports demonstrating that the CHOP transcrip-
tion factor, produced by PERK signaling, actively promotes
apoptosis in vitro and in vivo [3,13,15,16]. How can this
proapoptotic capacity of PERK signaling be reconciled with its
ability to enhance cell survival in the face of protein misfolding
[9,10,11,12]? Our findings suggest that the duration and/or
strength of PERK signaling may determine whether cytoprotective
or proapoptotic outcomes predominate. In our model, transient
PERK signaling protects cells by temporarily dampening cellular
protein synthesis and thus reducing misfolded protein levels in the
ER. Transient PERK signaling may also be insufficient to induce
CHOP levels to proapoptotic threshholds, given Chop’s inherent
Figure 2. Selective and specific activation of IRE1 signaling. (A)
Parental wild-type and transgenic HEK293 cells expressing the
Ire1[I642G], or Ire1[I642G/K599A] allele were treated for the indicated
times with 1NM-PP1 (1 mM), and wild-type cells were treated for
4 hours with thapsigargin (tg) (300 nM). IRE1[I642G] and IRE1[I642G/
K599A] protein was detected by immunoblotting for the FLAG epitope.
GAPDH levels were assessed as a protein loading control. Xbp1 mRNA
splicing was determined by RT-PCR. The unspliced (u) and spliced (s)
Xbp1 mRNA products are indicated as labeled. ERdj4 mRNA levels were
measured by quantitative PCR, normalized to Rpl19 mRNA levels, and
are shown relative to levels in untreated cells. (B) Parental wild-type and
transgenic HEK293 cells expressing the Ire1[I642G] or Ire1[I642G/K599A]
alleles were treated for the indicated times with 1NM-PP1 (1 mM); wild-
type cells were also treated for 4 hours with thapsigargin (tg) (300 nM).
ATF4 protein was detected by immunoblotting. GAPDH levels were
assessed as a protein loading control.
doi:10.1371/journal.pone.0004170.g002
PERK, IRE1, and Apoptosis
PLoS ONE | www.plosone.org
3
January 2009 | Volume 4 | Issue 1 | e4170


# Página 4

mRNA and protein instability [30]. However, persistent PERK
signaling could ultimately impair cell viability if extended
translational inhibition interrupted the generation of proteins vital
for cellular homeostasis. Persistent PERK signaling could also lead
to the accumulation of sufficient CHOP to drive cell death.
Intriguingly, in some cell types, CHOP directly induces the
transcription of Bim, a proapoptotic member of the BCL2 protein
family that directly elicits cell death by permeabilizing the
mitochondrial outer membrane [31]. A PERK-CHOP-BIM
signaling axis could link chronic protein misfolding in the ER to
activation of the intrinsic apoptosis machinery in the mitochon-
dria. Additional parallel proapoptotic signaling pathways must also
exist given the continued sensitivity of Perk and Chop null cells to
ER protein misfolding [9,13].
Can IRE1 also transmit apoptotic signals from the ER? While we
demonstrate a cytoprotective function for IRE1 signaling through its
RNAse activity, in mammalian cells, IRE1 has acquired additional
properties independent of splicing that include activation of the JNK
signaling pathway and selective biochemical interactions with the
BAK and BAX proteins of the BCL2 family of apoptotic regulators
[6,32]. The JNK signaling pathway and BCL2 proteins are key
regulators of cell survival and apoptosis in response to numerous
stimuli [23,33]. Although the consequences of their interactions with
the IRE1 signaling pathway on cell survival are unknown, they raise
the possibility that IRE1 employs multiple downstream modules
besides XBP1 generation to regulate cell fate after activation by
protein misfolding. IRE1’s oligomerization status has recently been
shown to regulate its RNAse activity [34]. Investigating the effect of
IRE1 polymerization status on JNK and BAX/BAK activity may
shed additional insight into IRE1’s effects on cell survival.
Divergent effects of persistent PERK and IRE1 signaling on cell
proliferation and survival may also underlie the phenotypes
observed in several pathologic and physiological situations in vivo.
Mice on high-fat diets developed hepatocyte steatosis, accompa-
nied by inflammation and PERK activation, suggesting a link
between PERK signaling and cellular dysfunction [35,36]. By
Figure 3. Sustained Perk signaling impairs cell proliferation. (A) Parental wild-type and isogenic HEK293 cells expressing Fv2E-Perk,
Ire1[I642G], or Ire1[I642G/K599A] alleles were treated with tunicamycin (5 mg/ml), 1NM-PP1 (1 mM), or AP20187 (2 nM), videographed for 48 hours, and
frames from indicated time points are shown. Magnification bar, 125 mm. (B) Parental wild-type and transgenic HEK293 cells expressing the Fv2E-Perk
allele were treated with AP20187 (2 nM), counted, and are shown relative to numbers of mock-treated cells at the indicated times. (C) Parental wild-
type and transgenic HEK293 cells expressing Ire1[I642G] or Ire1[I642G/K599A] alleles were treated with 1NM-PP1 (1 mM), counted, and are shown
relative to numbers of mock-treated cells at the indicated times.
doi:10.1371/journal.pone.0004170.g003
Figure 4. Sustained Perk signaling promotes apoptosis. Parental
wild-type and isogenic HEK293 cells expressing Ire1[I642G] or Fv2E-Perk
were treated with thapsigargin (300 nM); 1NM-PP1 (1 mM); or AP20187
(2 nM) for the indicated times. Cleaved PARP protein was assessed by
immunoblot. GAPDH protein levels served as a loading control.
doi:10.1371/journal.pone.0004170.g004
PERK, IRE1, and Apoptosis
PLoS ONE | www.plosone.org
4
January 2009 | Volume 4 | Issue 1 | e4170


# Página 5

contrast, selective expression of spliced XBP1 protein in B-cells
dramatically enhanced cell numbers, leading to a multiple
myeloma-like phenotype [37], consistent with the ability of IRE1’s
RNAse function to promote cell proliferation and survival.
Pharmacological modulation of PERK or IRE1 signaling could
provide new approaches to treat diseases associated with ER stress.
Materials and Methods
Molecular Biology
Generation of the AP20187 dimerizable Fv2E-Perk allele and
1NM-PP1 sensitized Ire1[I642G] allele has been previously described
(Lu et al., 2000; Lin et al., 2007). To construct the Ire1[I642G/
K599A] allele, QuikChange site-directed mutagenesis (Stratagene,
San Diego, CA) was used to insert a lysine to alanine missense
mutation in the Ire1[I642G] allele at amino acid position 599.
RT-PCR analysis of Xbp1 mRNA splicing was performed as
previously described (Lin et al., 2007). Primers used for quantita-
tive PCR analysis included: Fv2E-Perk mRNA, 59- TGAGTGT-
GGGTCAGAGAGCCAAAC-39 and 59- ACGGAGTCGTATT-
TACTTTCAGTC-39; human Rpl19 mRNA, 59-ATGTATCA-
CAGCCTGTACCTG–39 and 59-TTCTTGGTCTCTTCCTC-
CTTG-39; human Chop mRNA, 59-ACCAAGGGAGAACCAG-
GAAACG-39 and 59-TCACCATTCGGTCAATCAGAGC-39;
human Gadd34 mRNA, 59- CCTCTACTTCTGCCTTGTCTC-
CAG -39 and 59- TTTTCCTCCTTCTCCTCGGACG -39; and
human ERdj4 mRNA, 59- TGGTGGTTCCAGTAGACAA-
AGG-39
and
59-
CTTCGTTGAGTGACAGTCCTGC-39.
Quantitative PCR was performed using a MJ Opticon 2 DNA
Engine (Bio-Rad, Hercules, CA) as previously described (Lin et al.,
2007).
Protein Analysis
The following antibodies and dilutions were used for Western
analyses: anti-FKBP at 1:1000 (Affinity BioReagents, Golden, CO);
anti-eIF2a at 1:2000 (Cell Signaling, Natick MA); anti-phospho-
eIF2a at 1:500 (Cell Signaling, Natick, MA); anti-ATF4 at 1:2000
(Santa Cruz Biotechnologies, Santa Cruz, CA); anti-GAPDH at
1:10000000 (AbCAM, Cambridge, MA); anti-FLAG at 1:5000
(Sigma, St. Louis, MO); and anti-PARP at 1:2000 (Cell Signaling,
Natick, MA).
Cell Culture
HEK293 cell lines were maintained at 37uC, 5% CO2 in DMEM
media supplemented with fetal calf serum, glutamine, and antibiotics
(Invitrogen, San Diego, CA). Tunicamycin and thapsigargin were
obtained from Calbiochem EMD Bioscience Inc. (Darmstadt,
Germany). AP20187 was provided by Ariad Pharmaceuticals
(Cambridge, MA) and used as directed. 1NM-PP1 was used as
previously
described
[18].
The
Fv2E-Perk,
Ire1[I642G],
and
Ire1[I642G/K599A] alleles were integrated into HEK293 cells
bearing frt sites as previously described (Lin et al., 2007). Multiple
independent isogenic clones were analyzed with identical findings.
The CHO cell line bearing Fv2E-Perk has been previously
described [26]. To obtain resistant cells, CHO cells bearing Fv2E-
Perk were plated at clonal density and grown for 10 days in
100 nM AP20187 (and 3 mg/ml puromycin to enforce expression
of the Fv2E-Perk retroviral transgene). Multiple resistant clones
were identified under such conditions and individually expanded
for Fv2E-PERK protein expression analysis.
Cell Microscopy, Image Acquisition, and Cell Counts
Wild-type or isogenic HEK293 cells bearing Fv2E-Perk, Ir-
e1[I642G], or Ire1[I642G/K599A] alleles were plated at densities of
75000 cells/ml and live-cell imaging was performed using an
inverted microscope (Nikon TE2002E2) with a 106 0.3NA
objective and a cooled charge-coupled device camera (Coolsnap
HQ2, Photometrics) in a sealed humidified 5% CO2, 37 C
chamber. Images were acquired at 5-minute intervals for 48 hours
after application of tunicamycin (5 mg/ml), 1NM-PP1 (1 mM),
AP20187 (2 nM), or dimethylformamide solvent using Nikon
Imaging Systems Elements 2.3 software. Images were exported as
TIFF files into ImageJ software to compile into video files and to
capture frames for cell counts. Three to six independent imaging
experiments were conducted for each condition, and representative
videos are shown.
Supporting Information
Figure S1
Titration of AP20187 on HEK293 cells expressing
Fv2E-Perk. (A) Transgenic HEK293 cells were treated for 4 hours
with thapsigargin (300 nM) or the indicated concentrations of
AP20187. Fv2E-PERK and ATF4 proteins were detected by
immunoblotting. Chop mRNA levels were measured by quanti-
tative PCR, normalized to Rpl19 mRNA levels, and shown
relative to levels in mock-treated cells. (B) Transgenic HEK293
cells were treated for 4 hours with thapsigargin (300 nM) or the
indicated concentrations of AP20187. Xbp1 mRNA splicing was
assesed by RT-PCR. The unspliced (u) and spliced (s) Xbp1
mRNA products are indicated as labeled.
Found at: doi:10.1371/journal.pone.0004170.s001 (1.67 MB EPS)
Figure S2
Effect of tunicamycin, 1NM-PP1, or AP20187 on
PARP processing in HEK293 cells. Parental wild-type HEK293
cells were treated for the indicated times with tunicamycin (tu) (5
IˆJg/ml), 1NM-PP1 (1 IˆJM), or AP20187 (2 nM). Cleaved
PARP protein was assessed by immunoblot. GAPDH protein
levels served as a loading control.
Found at: doi:10.1371/journal.pone.0004170.s002 (1.94 MB EPS)
Figure S3
Loss of Fv2E-PERK restores cell viability in CHO
cells. Fv2E-PERK protein (+/2 phosphorylation) was examined
by immunoblotting in parental CHO cells expressing stably-
integrated Fv2E-Perk and 6 clonal derivatives that grew in the
presence of AP20187 (100 nM). Ponceau S staining of the
immunoblot revealed equivalent protein levels and served as a
loading control (data not shown). Where indicated, cells were
exposed to AP20187 (100 nM) for 30 minutes.
Found at: doi:10.1371/journal.pone.0004170.s003 (0.63 MB EPS)
Video S1
HEK293 cells treated with mock solvent (left frame) or
tunicamycin (right frame) for 48 hours.
Found
at:
doi:10.1371/journal.pone.0004170.s004
(7.81
MB
MOV)
Video S2
HEK293 cells expressing Fv2E-PERK treated with
mock solvent (left frame) or AP20187 (right frame) for 48 hours.
Found at: doi:10.1371/journal.pone.0004170.s005 (10.37 MB
MPG)
Video S3
HEK293 cells expressing IRE1[I642G] treated with
mock solvent (left frame) or 1NM-PP1 (right frame) for 48 hours.
Found at: doi:10.1371/journal.pone.0004170.s006 (10.15 MB
MPG)
Acknowledgments
We thank B. Farese, M. Karin, and B. Yen for helpful comments. We
thank K. Thorn. F. Sanchez, and the Nikon Imaging Center for technical
assistance. We thank C. Zhang and K. Shokat for generous provision of
1NM-PP1.
PERK, IRE1, and Apoptosis
PLoS ONE | www.plosone.org
5
January 2009 | Volume 4 | Issue 1 | e4170


# Página 6

Author Contributions
Conceived and designed the experiments: JHL HL YZ DR PW. Performed
the experiments: JHL HL YZ. Analyzed the data: JHL HL YZ DR PW.
Contributed reagents/materials/analysis tools: JHL YZ DR PW. Wrote
the paper: JHL PW.
References
1. Ron D, Walter P (2007) Signal integration in the endoplasmic reticulum
unfolded protein response. Nat Rev Mol Cell Biol 8: 519–529.
2. Harding HP, Zhang Y, Ron D (1999) Protein translation and folding are
coupled by an endoplasmic-reticulum-resident kinase. Nature 397: 271–274.
3. Harding HP, Novoa I, Zhang Y, Zeng H, Wek R, et al. (2000) Regulated
translation initiation controls stress-induced gene expression in mammalian cells.
Mol Cell 6: 1099–1108.
4. Zhou D, Palam LR, Jiang L, Narasimhan J, Staschke KA, et al. (2008)
Phosphorylation of eIF2 directs ATF5 translational control in response to diverse
stress conditions. J Biol Chem 283: 7064–7073.
5. Calfon M, Zeng H, Urano F, Till JH, Hubbard SR, et al. (2002) IRE1 couples
endoplasmic reticulum load to secretory capacity by processing the XBP-1
mRNA. Nature 415: 92–96.
6. Urano F, Wang X, Bertolotti A, Zhang Y, Chung P, et al. (2000) Coupling of
stress in the ER to activation of JNK protein kinases by transmembrane protein
kinase IRE1. Science 287: 664–666.
7. Nishitoh H, Matsuzawa A, Tobiume K, Saegusa K, Takeda K, et al. (2002)
ASK1 is essential for endoplasmic reticulum stress-induced neuronal cell death
triggered by expanded polyglutamine repeats. Genes Dev 16: 1345–1355.
8. Lee AH, Iwakoshi NN, Glimcher LH (2003) XBP-1 regulates a subset of
endoplasmic reticulum resident chaperone genes in the unfolded protein
response. Mol Cell Biol 23: 7448–7459.
9. Harding HP, Zhang Y, Bertolotti A, Zeng H, Ron D (2000) Perk is essential for
translational regulation and cell survival during the unfolded protein response.
Mol Cell 5: 897–904.
10. Scheuner D, Song B, McEwen E, Liu C, Laybutt R, et al. (2001) Translational
control is required for the unfolded protein response and in vivo glucose
homeostasis. Mol Cell 7: 1165–1176.
11. Lu PD, Jousse C, Marciniak SJ, Zhang Y, Novoa I, et al. (2004) Cytoprotection
by pre-emptive conditional phosphorylation of translation initiation factor 2.
EMBO J 23: 169–179.
12. Boyce M, Bryant KF, Jousse C, Long K, Harding HP, et al. (2005) A selective
inhibitor of eIF2alpha dephosphorylation protects cells from ER stress. Science
307: 935–939.
13. Zinszner H, Kuroda M, Wang X, Batchvarova N, Lightfoot RT, et al. (1998)
CHOP is implicated in programmed cell death in response to impaired function
of the endoplasmic reticulum. Genes Dev 12: 982–995.
14. Southwood CM, Garbern J, Jiang W, Gow A (2002) The unfolded protein
response modulates disease severity in Pelizaeus-Merzbacher disease. Neuron
36: 585–596.
15. Oyadomari S, Koizumi A, Takeda K, Gotoh T, Akira S, et al. (2002) Targeted
disruption of the Chop gene delays endoplasmic reticulum stress-mediated
diabetes. J Clin Invest 109: 525–532.
16. Pennuto M, Tinelli E, Malaguti M, Del Carro U, D’Antonio M, et al. (2008)
Ablation of the UPR-mediator CHOP restores motor function and reduces
demyelination in Charcot-Marie-Tooth 1B mice. Neuron 57: 393–405.
17. Lange PS, Chavez JC, Pinto JT, Coppola G, Sun CW, et al. (2008) ATF4 is an
oxidative stress-inducible, prodeath transcription factor in neurons in vitro and
in vivo. J Exp Med 205: 1227–1242.
18. Lin JH, Li H, Yasumura D, Cohen HR, Zhang C, et al. (2007) IRE1 signaling
affects cell fate during the unfolded protein response. Science 318: 944–949.
19. Han D, Upton JP, Hagen A, Callahan J, Oakes SA, et al. (2008) A kinase
inhibitor activates the IRE1alpha RNase to confer cytoprotection against ER
stress. Biochem Biophys Res Commun 365: 777–783.
20. Lee AH, Iwakoshi NN, Anderson KC, Glimcher LH (2003) Proteasome
inhibitors disrupt the unfolded protein response in myeloma cells. Proc Natl
Acad Sci U S A 100: 9946–9951.
21. Lee AH, Chu GC, Iwakoshi NN, Glimcher LH (2005) XBP-1 is required for
biogenesis of cellular secretory machinery of exocrine glands. EMBO J 24:
4368–4380.
22. Nishitoh H, Kadowaki H, Nagai A, Maruyama T, Yokota T, et al. (2008) ALS-
linked mutant SOD1 induces ER stress- and ASK1-dependent motor neuron
death by targeting Derlin-1. Genes Dev 22: 1451–1464.
23. Barr RK, Bogoyevitch MA (2001) The c-Jun N-terminal protein kinase family of
mitogen-activated protein kinases (JNK MAPKs). Int J Biochem Cell Biol 33:
1047–1063.
24. Ventura JJ, Hubner A, Zhang C, Flavell RA, Shokat KM, et al. (2006) Chemical
genetic analysis of the time course of signal transduction by JNK. Mol Cell 21:
701–710.
25. Sakurai T, Maeda S, Chang L, Karin M (2006) Loss of hepatic NF-kappa B
activity enhances chemical hepatocarcinogenesis through sustained c-Jun N-
terminal kinase 1 activation. Proc Natl Acad Sci U S A 103: 10544–10551.
26. Lu PD, Harding HP, Ron D (2004) Translation reinitiation at alternative open
reading frames regulates gene expression in an integrated stress response. J Cell
Biol 167: 27–33.
27. Novoa I, Zeng H, Harding HP, Ron D (2001) Feedback inhibition of the
unfolded protein response by GADD34-mediated dephosphorylation of
eIF2alpha. J Cell Biol 153: 1011–1022.
28. Tirasophon W, Lee K, Callaghan B, Welihinda A, Kaufman RJ (2000) The
endoribonuclease activity of mammalian IRE1 autoregulates its mRNA and is
required for the unfolded protein response. Genes Dev 14: 2725–2736.
29. Oliver FJ, de la Rubia G, Rolli V, Ruiz-Ruiz MC, de Murcia G, et al. (1998)
Importance of poly(ADP-ribose) polymerase and its cleavage in apoptosis.
Lesson from an uncleavable mutant. J Biol Chem 273: 33533–33539.
30. Rutkowski DT, Arnold SM, Miller CN, Wu J, Li J, et al. (2006) Adaptation to
ER stress is mediated by differential stabilities of pro-survival and pro-apoptotic
mRNAs and proteins. PLoS Biol 4: e374.
31. Puthalakath H, O’Reilly LA, Gunn P, Lee L, Kelly PN, et al. (2007) ER stress
triggers apoptosis by activating BH3-only protein Bim. Cell 129: 1337–1349.
32. Hetz C, Bernasconi P, Fisher J, Lee AH, Bassik MC, et al. (2006) Proapoptotic
BAX and BAK modulate the unfolded protein response by a direct interaction
with IRE1alpha. Science 312: 572–576.
33. Danial NN (2007) BCL-2 family proteins: critical checkpoints of apoptotic cell
death. Clin Cancer Res 13: 7254–7263.
34. Kimata Y, Ishiwata-Kimata Y, Ito T, Hirata A, Suzuki T, et al. (2007) Two
regulatory steps of ER-stress sensor Ire1 involving its cluster formation and
interaction with unfolded proteins. J Cell Biol 179: 75–86.
35. Monetti M, Levin MC, Watt MJ, Sajan MP, Marmor S, et al. (2007)
Dissociation of hepatic steatosis and insulin resistance in mice overexpressing
DGAT in the liver. Cell Metab 6: 69–78.
36. Oyadomari S, Harding HP, Zhang Y, Oyadomari M, Ron D (2008)
Dephosphorylation of translation initiation factor 2alpha enhances glucose
tolerance and attenuates hepatosteatosis in mice. Cell Metab 7: 520–532.
37. Carrasco DR, Sukhdeo K, Protopopova M, Sinha R, Enos M, et al. (2007) The
differentiation and stress response factor XBP-1 drives multiple myeloma
pathogenesis. Cancer Cell 11: 349–360.
PERK, IRE1, and Apoptosis
PLoS ONE | www.plosone.org
6
January 2009 | Volume 4 | Issue 1 | e4170
