# Página 1

RESEARCH ARTICLE
A Circulating microRNA Signature Predicts
Age-Based Development of Lymphoma
Afshin Beheshti1, Charles Vanderburg2, J. Tyson McDonald3, Charusheila Ramkumar4,
Tatenda Kadungure4, Hong Zhang4, Ronald B. Gartenhaus5, Andrew M. Evens1*
1 Division of Hematology/Oncology, Molecular Oncology Research Institute, Tufts Medical Center, Boston,
Massachusetts, United States of America, 2 Harvard NeuroDiscovery Center, Massachusetts General
Hospital, Boston, Massachusetts, United States of America, 3 Cancer Research Center, Hampton University,
Hampton, Virginia, United States of America, 4 Department of Cell Biology and Development, University of
Massachusetts Medical School, Worcester, Massachusetts, United States of America, 5 Marlene & Stewart
Greenebaum Cancer Center, Department of Medicine, University of Maryland, Baltimore, Maryland, United
States of America
* AEvens@tuftsmedicalcenter.org
Abstract
Extensive epidemiological data have demonstrated an exponential rise in the incidence of
non-Hodgkin lymphoma (NHL) that is associated with increasing age. The molecular etiol-
ogy of this remains largely unknown, which impacts the effectiveness of treatment for
patients. We proposed that age-dependent circulating microRNA (miRNA) signatures in the
host influence diffuse large B cell lymphoma (DLBCL) development. Our objective was to
examine tumor development in an age-based DLBCL system using an inventive systems
biology approach. We harnessed a novel murine model of spontaneous DLBCL initiation
(Smurf2-deficient) at two age groups: 3 and 15 months old. All Smurf2-deficient mice
develop visible DLBCL tumor starting at 15 months of age. Total miRNA was isolated from
serum, bone marrow and spleen and were collected for all age groups for Smurf2-deficient
mice and age-matched wild-type C57BL/6 mice. Using systems biology techniques, we
identified a list of 10 circulating miRNAs being regulated in both the spleen and bone marrow
that were present in DLBCL forming mice starting at 3 months of age that were not present
in the control mice. Furthermore, this miRNA signature was found to occur circulating in the
blood and it strongly impacted JUN and MYC oncogenic signaling. In addition, quantification
of the miRNA signature was performed via Droplet Digital PCR technology. It was discov-
ered that a key miRNA signature circulates throughout a host prior to the formation of a
tumor starting at 3 months old, which becomes further modulated by age and yielded calcu-
lation of a ‘carcinogenic risk score’. This novel age-based circulating miRNA signature may
potentially be leveraged as a DLBCL risk profile at a young age to predict future lymphoma
development or disease progression as well as for potential innovative miRNA-based tar-
geted therapeutic strategies in lymphoma.
PLOS ONE | DOI:10.1371/journal.pone.0170521
January 20, 2017
1 / 16
a1111111111
a1111111111
a1111111111
a1111111111
a1111111111
OPEN ACCESS
Citation: Beheshti A, Vanderburg C, McDonald JT,
Ramkumar C, Kadungure T, Zhang H, et al. (2017)
A Circulating microRNA Signature Predicts Age-
Based Development of Lymphoma. PLoS ONE 12
(1): e0170521. doi:10.1371/journal.pone.0170521
Editor: Jianjun Zhao, Cleveland Clinic, UNITED
STATES
Received: October 6, 2016
Accepted: January 5, 2017
Published: January 20, 2017
Copyright: © 2017 Beheshti et al. This is an open
access article distributed under the terms of the
Creative Commons Attribution License, which
permits unrestricted use, distribution, and
reproduction in any medium, provided the original
author and source are credited.
Data Availability Statement: All relevant data are
within the paper and its Supporting Information
files.
Funding: This research was supported in part by
R01 CA164311. The funder had no role in study
design, data collection and analysis, decision to
publish, or preparation of the manuscript. There
was no additional external funding received for this
study.
Competing Interests: The authors have declared
that no competing interests exist.


# Página 2

Introduction
Diffuse large B-cell lymphoma (DLBCL) is the most common form of non-Hodgkin lym-
phoma (NHL), accounting for approximately one-third of patients diagnosed in the United
States [1, 2]. Although DLBCL is curable in the majority of patients, approximately 35–40% of
patients die due to disease progression, while acute and late toxicities remain an issue among
treated patients [1, 2]. Detection and treatment options for DLCBL are typically developed by
observational clinical studies rather than measurable biological differences [3, 4]. This has
resulted in a general lack of precision medicine approaches to date in current DLBCL thera-
peutic paradigms [3, 5]. Varied molecular factors, however, are emerging as potential prognos-
tic and therapeutic targets in DLBCL [2, 6]. A specific transcription factor, JUN, was shown to
be frequently activated in DLBCL and highly upregulated in a large number of genes and sig-
nificantly contributed to DLBCL growth through interaction with the microenvironment [7].
Additionally, we recently showed in an analysis utilizing The Cancer Genome Atlas (TCGA)
that JUN impacted older DLBCL patients [8]. In general, continued knowledge is needed to
identify specific molecular changes and potential actionable pathways for prognosis and thera-
peutic targets in DLBCL.
MicroRNAs (miRNAs) are small non-coding RNAs that impact post-transcriptional gene
expression and are increasingly being recognized in cancer, including NHL, as important in
pathogenesis, prognosis, and therapy [9–14]. Each miRNA can target hundreds of mRNAs,
which predicts that over half of the existing human transcriptome is regulated by miRNAs
[15, 16]. Not only do miRNAs impact the transcriptome, they are now known to target and
regulate proteins and DNA [17, 18]. Recent studies have started to implicate miRNAs in driv-
ing DLBCL progression [19–21], but which specific miRNA signatures impact DLBCL devel-
opment or progression remains to be fully delineated. MicroRNAs have also been implicated
with age. Evidence suggests a tissue-specific coordinated pool of miRNAs contribute to the
“hallmarks of aging” [22]. Overlap exists between the miRNA signatures in DLBCL and age
related miRNAs, but little is reported on how all these factors uniformly effect DLBCL devel-
opment, progression, and survival of patients. Further, the impact of where and how the miR-
NAs affecting these factors is not understood. Recent evidence showed distinct miRNA
signatures in the blood that arises from tumor burden [10, 23, 24]. These circulating miRNAs
are highly stable, resistant to degradation, and have potential to be used as a non-invasive
novel therapeutic strategy [23, 25]. Additionally, there have also been distinct circulating miR-
NAs associated with age-related changes that can impact a variety of diseases included DLBCL
[26–28]. Single circulating miRNAs are an attractive minimally invasive tool for potential use
as biomarkers for lymphoma detection [9–11, 13, 20, 24, 29–31].
We hypothesized that a single miRNA is not sufficient to predict DLBCL development or to
determine a risk profile; instead, a more complete multi-miRNA signature likely needs to be
identified to fully detect and potentially target the relevant complex and interacting pathways
involved with tumor initiation and development. Leveraging a novel spontaneous DLBCL
murine model with Smurf2 deficiency, we were able to identify an age-based key functional
circulating miRNA signature that occurs in the blood. This key circulating miRNA signature
consists of ten miRNAs (let-7c, let-7b, miR-15a, miR-18a, miR-27a, miR-155, miR-24, miR-
130a, miR-10b, and miR-497), which were responsible for DLBCL initiation and was present
prior to the formation of visible tumor. Quantification of this miRNA signature was done
through a cutting edge technology known as droplet digital PCR (ddPCR). Further systems
biology analyses revealed that this miRNA signature impacted oncogenic factors related to
JUN and MYC and other important functions associated with cancer cell growth.
miRNA Predictions for Lymphomagenesis
PLOS ONE | DOI:10.1371/journal.pone.0170521
January 20, 2017
2 / 16


# Página 3

Materials and Methods
DLBCL spontaneous murine model and collection of blood
We utilized a novel spontaneously producing DLBCL mouse model known as a Smurf2-defi-
cient mice (Smurf2T/T, T for the gene-trapping allele) described in detail in previous publica-
tions [32, 33]. Age matched C57BL/6 mice were used as wild-type mice and controls. Spleen
tissue and bone were harvested using previously described methodology after mice were sacri-
ficed from five Smurf2T/T and wild-type two month old mice [32, 33]. 100 to 200 μl of periph-
eral blood was harvested from the mice using facial vein blood collection. The blood is allowed
to clot for 30 min at room temperature and then centrifuged at 1000×g for 10 min. The serum
is removed into a new tube and stored at -80˚C until use. These biological replicates were used
for each condition: n = 11 young Smurf2T/T mice, n = 12 young wild-type mice, n = 8 old
Smurf2T/T mice, and n = 4 old wild-type mice. Young mice are considered to be ~3.5 months
old and old mice are ~15 months old. All animal studies were conducted at University of Mas-
sachusetts Medical School by Dr. Hong Zhang. At the time of blood collection, all mice were
healthy and none of them had visible tumors. For tumors collected for the westerns, mice were
sacrificed using standard protocols (isoflurane and cervical dislocation) as referenced in our
previous publication [32]. Additionally, spleen and bone marrow were harvested post-mortem
from other mice and sacrificed using the same methodology. This study was carried out in
strict accordance with the recommendations in the Guide for the Care and Use of Laboratory
Animals of the National Institutes of Health. The protocol was approved by the Institutional
Animal Care and Use Committee (IACUC) at University of Massachusetts Medical School
with protocol number A-1758-15, approved on 4/1/2015. All mice were housed in a fully
accredited Animal Care facility and the animal handling and procedures were approved by
University of Massachusetts Medical School IACUC.
miRNA expression profiling on mouse tissue
Expression profiling of miRNA was done on spleen and bone marrow tissue for Smurf2-defi-
cient mice (n = 5) and wild-type (n = 5) at 2 months of age. Microfluidic μParaflo1 technology
microarray platform (LC Sciences, Houston, TX) was used to obtain the miRNA expression
profile. Preprocessing of the miRNA signal was done by standard background subtraction and
normalization. The detectable miRNAs were considered by the following two conditions: 1.
signal intensity higher than 3×(background standard deviation) and 2. Coefficient of variation
(CV) <0.5. CV was calculated by the ratio of standard deviation to signal intensity. When
repeating probes are present on an array, the miRNA was listed as detectable only if the signals
from at least 50% of the repeating probes are above detection level. Statistically relevant miR-
NAs were considered with p-values<0.05. Fold changes were determined between the
Smurf2-deficient mice and wild-type mice and the average values per group were determined.
Network representation of how the miRNAs interact and impact on key transcriptional factors
was determined with Ingenuity Pathway Analysis (IPA) software (Ingenuity Systems).
miRNA isolation
Isolation of circulating miRNAs from serum was done using the QIAgen miRNeasy Mini Kit
(QIAgen, CA). Previously published studies revealed that the miRNeasy Mini Kit provides the
best yield and highest quality of miRNA from serum [34]. Concentration and quality of miR-
NAs was quantified by Eppendorf Biophotometer 6131 Spectrophotometer (Eppendorf, Haup-
pauge, NY).
miRNA Predictions for Lymphomagenesis
PLOS ONE | DOI:10.1371/journal.pone.0170521
January 20, 2017
3 / 16


# Página 4

cDNA generation and Droplet Digital PCR
From the isolated miRNA, cDNA was first generated using the miRCURY LNA™Universal RT
microRNA PCR Universal cDNA Synthesis Kit II (Exiqon, Woburn, MA) using a concentra-
tion of 5ng/μl for the miRNA per sample. The miRNA was quantified using the Bio-Rad iCy-
cler Droplet Digital and Real Time PCR system (ddPCR) (Bio-Rad, Hercules, CA). A 1:20
dilution of the generated cDNA was used with the QX200™ddPCR EvaGreen Supermix (Bio-
Rad). The following miRCURY LNA™Universal RT microRNA PCR LNA™PCR primer sets
(Exiqon) were used with the EvaGreen Supermix: hsa-let-7c-5p, hsa-let-7b-5p, mmu-miR-
497-5p, hsa-miR-130a-3p, hsa-miR-18a-5p, hsa-miR-24-3p, hsa-miR-15a-5p, hsa-miR-27a-3p,
hsa-miR-10b-5p, and mmu-miR-155-5p. Droplets were generated using the QX200™Auto-
mated Droplet Generator (Bio-Rad). The plates were sealed with the PX1™PCR Plate Sealer
(Bio-Rad). With the C1000 Touch™Thermal Cycler with 96–Deep Well Reaction Module
(Bio-Rad) the following PCR reaction was used for all the primers except for hsa-miR-10b-5p
and mmu-miR-155-5p: 1 cycle 95˚C for 5 min, 40 cycles of 95˚C for 30 sec and 58˚C for 1 min
(annealing temperature), 1 cycle of 4˚C for 5 min, and 1 cycle of 90˚C for 5 min. Not all
miRNA primers sets for ddPCR will have the same annealing temperature, so optimizing the
annealing temperature is required for each primer set. For hsa-miR-10b-5p and mmu-miR-
155-5p primer sets the optimal annealing temperatures were 54˚C and 52˚C. The QX200™
Droplet Digital™PCR System (Bio-Rad) quantified the amount of miRNA for each primer set
per sample. QuantaSoft software (Bio-Rad) generated the data for each primer set and sample.
The same threshold setting was used for all samples per primer set. The concentration
(miRNA copies/μl) value generated by QuantaSoft was converted to miRNA copies/ng of
serum.
Predicted miRNA impact on Carcinogenesis, mRNA, and molecular
pathways
The impact of this miRNA signature on cancer risk was determined through the literature as
to how each miRNA impacts DLBCL (i.e., OncomiR or tumor suppressor). Log2 fold-change
values were calculated for each miRNA comparing Smurf2T/T to wild-type mice. The overall
impact was used to determine the global effect on tumor progression and cancer risk based on
the specific regulation of each gene. A Carcinogenic Risk Score (CRS) is calculated based on
these values to determine if there is a promoted risk for cancer (positive value) or inhibitory
risk for cancer (negative value) by summing the log2 fold-change values of oncomiRs and sub-
tracting this from the sum of log2 fold-change values of the tumor suppressors when compar-
ing either Smurf2T/T to wild-type mice or old to young mice.
CluePedia Cytoscape plugin [35] was used to predict the impact of the miRNA signature on
the transcriptome (or mRNA). A threshold of 60 mRNA targets was used when generating the
mRNA networks associated with the miRNAs. Additional pathway analysis was done using
Ingenuity Pathway Analysis (IPA) software (Ingenuity Systems). The log2 fold-change values
of the miRNAs comparing Smurf2 deficient to wild-type mice were used to predict through
IPA the impact on cancer related functions.
Western Blot
Total cell lysates were collected using RIPA buffer (50mM Tris-HCl, pH 7.5, 150mM NaCl, 1%
Triton X-100, 0.1% SDS, 0.5% deoxycholic acid and 0.02% sodium azide) with freshly added
complete protease inhibitors (Roche). Protein lysates (20 mg) were separated by SDS–PAGE
Criterion X-gel (Bio-Rad) and transferred to nitrocellulose membranes (GE Osmonics).
miRNA Predictions for Lymphomagenesis
PLOS ONE | DOI:10.1371/journal.pone.0170521
January 20, 2017
4 / 16


# Página 5

Immunoblots were visualized using a western lightening chemiluminescence detection kit
(Perkin Elmer). The primary antibody was Anti-c-JUN antibody [E254] (Abcam, Cambridge,
MA) and anti-GAPDH [FL-335] (Santa Cruz Biotechnology).
Results
A common miRNA signature is present in spleen and bone marrow of
Smurf2 deficient mice
A novel spontaneous DLBCL murine model was previously established by creating
Smurf2-deficient mice (Smurf2T/T, T for the gene-trapping allele) with C57BL/6 background
[32, 33, 36]. We leveraged this model to study how miRNAs impact tumor formation.
Smurf2T/T mice will begin to develop DLBCL starting at 15 months of age [33]. Even though
all mice will form DLBCL starting at this age, not all mice manifest DLBCL formation at the
same time. The variation in DLBCL formation can range from 15 months to as long as 20
months [33].
To investigate the impact of DLBCL initiation on miRNA expression, a profile was per-
formed on both spleen and bone marrow tissue from two month old Smurf2T/T and wild-type
mice (S1 and S2 Tables). We determined with Ingenuity Pathway Analysis (IPA) that the sig-
nificantly regulated miRNAs (P<0.05) had the most impact on both p53 and TGFβ1 in both
the spleen and bone marrow for the Smurf2T/T when compared to the wild-type mice (Fig 1A
and 1C). Further analysis led to a surprising discovery of a common miRNA signature being
regulated in the same direction between the Smurf2T/T and wild-type mice that is present
between both the bone marrow and spleen tissue (Fig 1B and 1D). This miRNA signature con-
sists of 10 miRNAs: miR-130, miR-27, miR-17, miR-10, miR-155, let-7a-5p, let-7, miR-24-3p,
miR-15, and miR-16-5p. We previously reported that during tumor progression (and possibly
tumor initiation), there is a systemic impact on the host influencing tumor dynamics and
molecular factors originating from the host that may be the true “drivers” for carcinogenesis
[37, 38]. These data begin to reveal universal drivers (i.e., miRNAs) in the host responsible for
DLBCL formation.
Circulating miRNA signature causing DLBCL formation
The presence of circulating miRNAs in the blood provides a putative mechanism for how a
common miRNA signature may impact different areas of the host resulting in DLBCL forma-
tion. To examine this theory, we collected peripheral blood from 3.5 months old (“young”) and
15 months old (“old”) Smurf2T/T and wild-type mice. These ages are roughly equivalent to a
20-year old human for young mice and 50-year old human for the old mice [37]. At the time
the serum was prepared none of the old mice had any noticeable tumors. These mice will even-
tually develop tumors. The expression of 10 common miRNAs predicted above was tested
using serum to observe if these miRNAs are circulating through the host. Since miRNAs can
have different aliases, the 10 miRNAs (Fig 1) are identified as the following for the rest of this
manuscript: let-7 = let-7b, let-7a-5p = let-7c, miR-10 = miR-10b, miR-130 = miR-130a, miR-
155 = miR-155, miR-27 = miR27a, miR-24-3p = miR-24, miR-17 = miR-18a, miR-15 = miR-
15a, and miR-16-5p = miR-497. Through the use Droplet Digital PCR (ddPCR), we were able
to quantify exact counts of circulating miRNA that exists in the serum at high resolution.
ddPCR has superior precision over other detection technologies to detect differential expres-
sion in miRNAs [39]. Additionally, ddPCR is more resilient to sample quality differences; has
a high tolerance to PCR amplification inefficiency; and requires small sample size, which
allows for high level of precision being able to capture all known sources of variation [39, 40].
miRNA Predictions for Lymphomagenesis
PLOS ONE | DOI:10.1371/journal.pone.0170521
January 20, 2017
5 / 16


# Página 6

Fig 1. Common miRNA signature in spleen and bone marrow from DLBCL-forming mice. miRNA array was performed on spleen and bone
marrow samples from 2 month old mice for both Smurf2 deficient (Smurf2T/T) and wild-type (WT) mice. A network generated through IPA of the
significantly regulated miRNAs with p-value < 0.05 is shown with the top two most impacted mRNAs (TGFβ1 and p53) for A) spleen tissue and C)
bone marrow comparing Smurf2-deficient mice with wild-type mice. The 10 common miRNAs present in both the B) spleen and the D) bone marrow
are shown as a network with TGFβ1 and p53. The specific miRNA expression values can be found in S1 and S2 Tables.
doi:10.1371/journal.pone.0170521.g001
miRNA Predictions for Lymphomagenesis
PLOS ONE | DOI:10.1371/journal.pone.0170521
January 20, 2017
6 / 16


# Página 7

From the small amounts of serum, a distinct miRNA signature was captured starting at
three months of age, long before actual tumor formation when comparing Smurf2T/T to wild-
type mice (Fig 2). All miRNAs, except miR-497, were increased in Smurf2T/T mice compared
with wild-type mice (Fig 2). Specifically, let-7c, miR-27a, miR-155, and miR-24 were signifi-
cantly increased, while the rest of the miRNAs had an increased trend. There was also a large
variation between Smurf2T/T mice for most miRNAs, while the wild-type mice exhibit a small
variation between individual mice. As stated before, this is due in part that although Smurf2T/T
mice start to form DLBCL ~15 months of age, not all mice will form tumors at the same time
[33]. In part, since no tumors were present at the time of blood collection for the old mice, we
believe that this signature may predict tumor development before it actually forms. This
miRNA signature is an indicator of a propensity for future DLBCL growth and can be thought
of a risk profile for DLBCL at a young age, which has not previously been described.
As the mice approach the age where the DLBCL can start formation (at ~15 months of age),
these miRNAs provide a different signature associated with tumor formation. For old mice,
miR-497 was significantly increased and miR-155 was significantly decreased for Smurf2T/T
compared to wild-type mice (Fig 3). The rest of the circulating miRNAs started to demonstrate
a decrease for Smurf2T/T compared to wild-type mice, with exception of miR-130a with mini-
mal change. As with the younger mice, there was a larger variation that occurred for Smurf2T/T
mice compared to wild-type mice. The miRNA signature associated with Smurf2T/T mice at
this age can be associated with a distinct marker or risk for the presence of DLBCL.
To investigate how the miRNA signature changes with age for mice that will develop
DLBCL, comparisons were made between young and old Smurf2T/T mice and compared to
the equivalent wild-type mice (Fig 4). Five out of the ten miRNAs (let-7c, miR-15a, miR-18a,
miR-24, and miR-130a) showed an increased amount of circulating miRNA with age for both
Smurf2T/T and wild-type mice (Fig 4). Interestingly, most of the wild-type mice had a signifi-
cant increase, while the Smurf2T/T mice only showed the trend. As stated above, the large vari-
ance in Smurf2T/T mice was due to the fact that not all mice formed DLBCL at the same time.
Thus, these 5 miRNAs were impacted with age regardless of DLBCL formation and the inter-
estingly age-related miRNA signature that they present should be explored further at a future
date.
Fig 2. Circulating miRNA signature from the serum of 3-month old mice comparing Smurf2T/T to wild type mice. miRNA was quantified by
ddPCR and exact miRNA copies/nanogram of serum was measured. The data is shown as both a heatmap of miRNA copies/nanogram for each
mouse and the average counts (bar plots). The error bars in the bar plots represents the standard error. The color gradient for the heatmap represents
the miRNA copies/nanogram which is shown as different colors in the hearmap.
doi:10.1371/journal.pone.0170521.g002
miRNA Predictions for Lymphomagenesis
PLOS ONE | DOI:10.1371/journal.pone.0170521
January 20, 2017
7 / 16


# Página 8

When identifying miRNA changes with age specifically related to Smurf2T/T that differ
from the wild-type, miR-27a was the only miRNA which was being suppressed in Smurf2T/T
with age, while increasing with age in wild-type age (Fig 4). In the literature, increases in miR-
27a have been associated with increased survival for DLBCL patients [12]. The suppression of
Fig 3. Circulating miRNA signature from the serum of 15-month old mice comparing Smurf2T/T to wild type mice. miRNA was quantified by
ddPCR and exact miRNA copies/ng of serum was measured. The data is shown as both a heatmap of miRNA copies/ng for each mouse and the
average counts (bar plots). The error bars in the bar plots represents the standard error. The color gradient for the heatmap represents the miRNA
copies/nanogram which is shown as different colors in the hearmap.
doi:10.1371/journal.pone.0170521.g003
Fig 4. Circulating miRNA signature changing with age in mice susceptible to DLBCL formation compared to wild-type mice. miRNA was
quantified by ddPCR and exact miRNA copies/ng of serum was measured. Average miRNA counts are compared between young and old mice for both
Smurf2T/T (purple color) and wild-type (orange) groups. The p-values are indicated on the plots and the error bars in the bar plots represents the standard
error.
doi:10.1371/journal.pone.0170521.g004
miRNA Predictions for Lymphomagenesis
PLOS ONE | DOI:10.1371/journal.pone.0170521
January 20, 2017
8 / 16


# Página 9

miR-27a in Smurf2T/T mice with age correlates with the formation of DLBCL and the overall
survival of the mice due to the tumor.
The impact of circulating miRNA on carcinogenic risk
A systems biology approach was implemented to determine the overall impact this miRNA
signature will have on cancer risk when comparing either Smurf2T/T to wild-type mice or
young to old mice. Each of the 10 miRNAs was labeled as a tumor promoter (also referred to
as an oncomiR) or a tumor suppressor. The impact of the miRNA was directly associated with
DLBCL as opposed to cancer in general, since depending on the cancer type, a miRNA can
either be considered as a tumor suppressor or oncomiR (Fig 5). For example, miR-27a for
DLBCL has been shown to have overall improved survival for DLBCL patients [12], while in
breast cancer, it has been associated as an oncomiR and with poor patient survival [41]. Addi-
tionally, let-7b has typically been thought to be a tumor suppressor in most cancers, but it has
been shown in DLBCL that let-7b is overexpressed [31, 42] and suggests that it may be consid-
ered an oncomiR in this cancer subtype. In addition to miR-27a and let-7b, the following miR-
NAs from the miRNA signature were considered to be tumor suppressors for DLBCL: miR-
15a [29, 32, 43, 44], let-7c [20, 23], miR-24 [12], and miR-497 [9, 45]. The following miRNAs
were categorized as oncomiRs for DLBCL: miR-10b [44], miR-155 [9, 19, 29, 44], let-7b [31,
42], miR-18a [1, 14, 41, 44], and miR-130a [44, 46]. A Carcinogenic Risk Score (CRS) was subse-
quently calculated and determines whether the overall risk for cancer will be promoted (if pos-
itive) or inhibited (if negative). It is predicted based on this miRNA signature that Smurf2T/T
mice for both young (Fig 5A) and old (Fig 5B) age groups have a high positive carcinogenic
risk score which indicates the high probability of cancer formation. Interestingly, this miRNA
signature was shown to slightly reduce the impact of cancer formation with age (Fig 5C),
which indicates that this miRNA signature is more dominant for DLBCL initiation and
Fig 5. Cancer risk predictions for the circulating miRNA signature. Based on lymphoma literature the miRNAs are classified as tumor promoters
(or OncomiRs) and tumor suppressors. Fold-change values are calculated for each miRNA comparing Smurf2T/T (SM2) to wild-type (WT) mice. A
Carcinogenic Risk Score (CRS) is calculated based on these values to determine if there is a promoted risk for cancer (positive value) or inhibitory risk
for cancer (negative value). A) Cancer risk is promoted for young Smurf2T/T mice compared to wild-type. B) Cancer risk is promoted for old Smurf2T/T
mice compared to wild-type. C) Cancer risk is inhibited for Smurf2T/T old mice compared to Smurf2T/T young mice. D) Cancer risk is slightly promoted
for wild-type old mice compared to wild-type young mice.
doi:10.1371/journal.pone.0170521.g005
miRNA Predictions for Lymphomagenesis
PLOS ONE | DOI:10.1371/journal.pone.0170521
January 20, 2017
9 / 16


# Página 10

formation rather than progression. For wild-type mice, there was an overall slight increase risk
of cancer based on this miRNA signature when comparing old with young mice (Fig 5D). This
indicates that there is an age-related increase risk of cancer.
The functional impact of circulating miRNA
Since the existence of this miRNA signature in the host predicted a strong risk of DLBCL, it is
important to identify the key functional drivers and targets for these miRNAs. The impact of
this miRNA signature on mRNAs was determined by constructing a network of miRNAs
interactions with mRNAs (Fig 6A). The miRNA signature network for Smurf2T/T mice versus
wild type, revealed MYC and JUN as key targets (Fig 6A & 6B). Both MYC and JUN are known
to play an important role in DLBCL development and progression [7, 47–49]. The impact on
MYC and JUN from this circulating miRNA signature on the entire host starting at 3 months
of age likely act as a trigger for eventual development of DLBCL. We have previously shown
that an increase in MYC expression is observed in spleens and livers from two month old and
also in the fully formed lymphomas from older mice when comparing Smurf2T/T to wild-type
C57BL/6 mice [32]. In this publication several methods involving qPCR and westerns were
implemented on the different tissue and age groups to demonstrate the increase of MYC. We
Fig 6. Impact of miRNA signature on transcriptional factors and cancer related function. A) For young mice it was determined that the key
miRNA signature has impact on MYC and JUN along with other mRNAs determined using CluePedia Cytoscape plugin. The miRNA-mRNA
interactions were determined from three different databases as indicated by the figure legend. B) Using Ingenuity Pathway Analysis (IPA) the
functional impact of the miRNA signature is shown for young mice comparing Smurf2T/T mice to wild-type (WT) mice. C) Using IPA we predicted the
impact of the miRNA signature on JUN (indicated by blue color being downregulated). MYC expression is known to be downregulated in Smurf2T/T
mice from previous publications [32]. D) Western for JUN protein expression on tumor samples from Smurf2T/T mice (lanes 2–7), spleen from wild-
type (WT) C57BL/6 mice for the same age the tumor samples (lane 1), and spleen from young wild-type mice (lane 8) and young Smurf2T/T mice
(SM2) (lane 9).
doi:10.1371/journal.pone.0170521.g006
miRNA Predictions for Lymphomagenesis
PLOS ONE | DOI:10.1371/journal.pone.0170521
January 20, 2017
10 / 16


# Página 11

predict using IPA predictive tools that JUN signaling should be downregulated due to the
impact of the miRNA signature (Fig 6C). Furthermore, we validated the expression of JUN in
the Smurf2T/T mice compared to the wild-type mice. Western blot revealed a decrease in JUN
expression in the fully formed lymphomas (from different but identical Smurf2T/T mice which
the miRNA signature was measured from) (Fig 6D). Additionally when observing JUN expres-
sion from the spleen of young mice, we saw no difference in JUN signaling when comparing to
wild-type (WT) mice (Fig 6D). These MYC and JUN validations confirm the miRNA predicted
impact on these DLBCL related genes.
Finally, using Ingenuity Pathway Analysis (IPA), we were able to predict the impact on sig-
nificantly regulated biological functions from this miRNA signature (Fig 6B). For young
Smurf2T/T mice compared with the wild type, we observed an activation of cancer related func-
tions such as cell proliferation in tumors; inflammation; and lymphoid cancer related func-
tions (Fig 6B). We also noted prominent activation in apoptosis and necrosis. Although cell
death related mechanisms were activated at this young age due to the miRNA signature, on
balance, the long-term impact of the pro-cancer related functions likely provided sufficient
stimulus and drive for the eventual formation of DLBCL.
Discussion
Understanding of the impact of miRNAs in cancer has greatly evolved since the first links to
cancer were observed in early 2000s [50, 51]. More specifically, impact of miRNA on lympho-
mas was initially underestimated with only a handful of miRNAs, including miR-155 [52, 53]
and a miR-16 and miR-15 combination [54], first thought to be the key players in B cell lym-
phomas. Since then, it has been discovered that a number of other miRNAs, including miR-17
[13, 19], miR-27 [19, 20, 41], miR-24 [19], miR-10 [19, 20], and let-7 [12, 19, 20], play an
important role in lymphoma biology. This has led to many possible biomarkers and targets for
cancer therapeutics typically using only a single miRNA agent for lymphoma [19, 55].
We employed a systems biology approach to determine a key miRNA signature, and more-
over, one that had the most prominent impact on DLBCL development. To explore this, we
leveraged a spontaneous DLBCL xenograft model system known as Smurf2T/T, whereby 100%
of mice develop detectable lymphoma starting at 15 months (and up to 24 months of age). We
identified 10 common circulating miRNAs that were expressed in the spleen and bone marrow
of mice as early as 2 months of age. For young Smurf2 deficient mice compared with wild type
mice, there was a clear predicted risk of cancer promotion due to the miRNA signature detec-
tion 15 months prior to tumor formation. This miRNA signature was further modulated at
older ages associated with the beginning of DLBCL tumor formation. When determining the
impact of the miRNA signature on DLBCL biology, it was discovered that these miRNAs had
the most prominent impact on genes, MYC [48, 49] and JUN [7, 47], with global impact on
cancer related functions starting at age several months before actual DLBCL formation. We
have previously shown that JUN is a key driver for DLBCL with aging [8]. In addition, MYC is
commonly known to be involved DLBCL progression [48]. We have shown in a previous
Smurf2T/T related publication that MYC expression is elevated in spleen and liver tissue for
Smurf2T/T mice started at two months of age and also highly expressed in the eventual forma-
tion of the tumor [32]. With the additional data provided in this manuscript showing the
decreased expression of JUN in the Smurf2T/T mice (Fig 6C & 6D), we have uncovered a possi-
ble mechanism for the driving oncogenic factors in DLBCL through the newly identified
miRNA signature. We showed herein that there possibly exists a new mechanism in which a
DLBCL associated circulating miRNA signature suppresses JUN signaling before tumor for-
mation while promoting MYC oncogenic factors. Through a simple minimally invasive
miRNA Predictions for Lymphomagenesis
PLOS ONE | DOI:10.1371/journal.pone.0170521
January 20, 2017
11 / 16


# Página 12

technique for detecting miRNA signature associated with DLBCL, we were able to predict the
oncogenic factors which are known to promote and cause DLBCL. This methodology could
conceivably have clinical applications in determining potential treatment strategies before the
actual formation for of the tumor.
Currently, early detection of DLBCL is done through non-specific blood markers [56] and/
or varied imaging modalities [57]. Current diagnostic methods are only able to detect DLBCL
once macroscopic tumor/disease has already developed. This novel miRNA signature could
conceivable be used for both determining an early risk profile for DLBCL before formation and
it also has potential for developing innovative targeted therapeutic strategies to treat DLBCL.
Recent studies have shown that miRNAs contained in our miRNA signature, such as miR-
155, are potentially involved in DLBCL development [58, 59]. Although the extant literature is
promising, these studies do not leverage the important consideration of a signature of miRNAs
working together concurrently to impact both the development and progression of DLBCL.
Furthermore, as shown here, the miRNA signature appeared to mechanistically contribute to
DLBCL development even at an early age/point, before frank DLBCL formation. Additionally,
it has to be noted that specific miRNAs do not have a universal impact for all cancer types. For
example let-7b is generally considered to be a tumor suppressor [60], but when specifically
researching let-7b for DLBCL we have found that it actually is associated with tumor promo-
tion and should be considered an oncomiR [31, 42]. Future studies involving specific cancer
subtypes should determine the specific impact of the miRNAs on that cancer.
From this early stage, the possibility of cancer prevention can also be implemented by
applying a combination of inhibitors for miRNAs (anti-miRs) and miRNA mimetics to miti-
gate cancer risk. Application of single miRNA related therapy is being studied as a treatment
platform [24, 30]. In other cancers using synthetic miRNAs, or mimetics, has demonstrated
efficacy with exciting potential for targeted therapy [61, 62]. A single miRNA treatment in
murine breast cancer models, such as let-7a or miR-16, resulted in improved survival [43, 63].
While this has shown promising results, it only reduced tumor growth without fully eradicat-
ing the tumor. Additionally, the use of anti-miRs to block specific oncomiRs has also shown
encouraging results in reduction of tumor growth [64, 65]. To date, therapeutic attempts via
modulation of miRNA have been done exclusively with single miRNA targets. It is likely that a
combination of miRNA mimetics and anti-miRs together or in combination with other ratio-
nal targeted therapeutics will be warranted for effective cell kill.
Through our newly discovered miRNA signature, strategic combinations of anti-miR and
RNA mimetics may be able to tip the balance from cancer being promoted to inhibited (Figs 5
and 6) by targeting entire pathways and functions regulated by these miRNAs responsible for
tumor promotion (Fig 6). Additional work with this miRNA signature should be employed in
human studies as potential correlative translational diagnostic assays to test whether it is
detectable in DLBCL patients, and moreover, if it is predictive of tumor development or pro-
gression. In addition, a future goal might include leveraging such human-specific miRNA sig-
nature antagonists in patients with existing DLBCL or with high-risk potential of lymphoma
development or progression. Collectively, the overall impact of this work has both short and
long-term potential for developing novel cancer biomarkers and potential innovative and
highly targeted therapeutic approaches in lymphoma.
Supporting Information
S1 Table. The mean miRNA expression values and p-values from spleen tissue from two
month old Smurf2-/- and wild-type mice.
(DOCX)
miRNA Predictions for Lymphomagenesis
PLOS ONE | DOI:10.1371/journal.pone.0170521
January 20, 2017
12 / 16


# Página 13

S2 Table. The mean miRNA expression values and p-values from bone marrow tissue from
two month old Smurf2-/- and wild-type mice.
(DOCX)
Acknowledgments
This research was supported in part by R01 CA164311 (AME).
Author Contributions
Conceptualization: AB AME.
Data curation: AB.
Formal analysis: AB.
Funding acquisition: AME.
Investigation: AB CV HZ CR TK.
Methodology: AB CV HZ.
Project administration: AB AME.
Resources: HZ AME CV.
Supervision: AB AME.
Validation: AB CV.
Visualization: AB.
Writing – original draft: AB.
Writing – review & editing: AB CV JTM HZ RBG AME.
References
1.
Skrabek P, Turner D, Seftel M. Epidemiology of non-Hodgkin lymphoma. Transfus Apher Sci. 2013;
49(2):133–8. doi: 10.1016/j.transci.2013.07.014 PMID: 23958141
2.
Martelli M, Ferreri AJ, Agostinelli C, Di Rocco A, Pfreundschuh M, Pileri SA. Diffuse large B-cell lym-
phoma. Crit Rev Oncol Hematol. 2013; 87(2):146–71. doi: 10.1016/j.critrevonc.2012.12.009 PMID:
23375551
3.
Sarkozy C, Coiffier B. Diffuse large B-cell lymphoma in the elderly: a review of potential difficulties. Clin
Cancer Res. 2013; 19(7):1660–9. doi: 10.1158/1078-0432.CCR-12-2837 PMID: 23339126
4.
Advani RH, Chen H, Habermann TM, Morrison VA, Weller EA, Fisher RI, et al. Comparison of conven-
tional prognostic indices in patients older than 60 years with diffuse large B-cell lymphoma treated with
R-CHOP in the US Intergroup Study (ECOG 4494, CALGB 9793): consideration of age greater than 70
years in an elderly prognostic index (E-IPI). Br J Haematol. 2010; 151(2):143–51. doi: 10.1111/j.1365-
2141.2010.08331.x PMID: 20735398
5.
Klapper W, Kreuz M, Kohler CW, Burkhardt B, Szczepanowski M, Salaverria I, et al. Patient age at diag-
nosis is associated with the molecular characteristics of diffuse large B-cell lymphoma. Blood. 2012;
119(8):1882–7. doi: 10.1182/blood-2011-10-388470 PMID: 22238326
6.
Zhang J, Grubor V, Love CL, Banerjee A, Richards KL, Mieczkowski PA, et al. Genetic heterogeneity of
diffuse large B-cell lymphoma. Proc Natl Acad Sci U S A. 2013; 110(4):1398–403. doi: 10.1073/pnas.
1205299110 PMID: 23292937
7.
Blonska M, Zhu Y, Chuang HH, You MJ, Kunkalla K, Vega F, et al. Jun-regulated genes promote inter-
action of diffuse large B-cell lymphoma with the microenvironment. Blood. 2015; 125(6):981–91. doi:
10.1182/blood-2014-04-568188 PMID: 25533033
miRNA Predictions for Lymphomagenesis
PLOS ONE | DOI:10.1371/journal.pone.0170521
January 20, 2017
13 / 16


# Página 14

8.
Beheshti A, Neuberg D, McDonald JT, Vanderburg CR, Evens AM. The Impact of Age and Sex in
DLBCL: Systems Biology Analyses Identify Distinct Molecular Changes and Signaling Networks. Can-
cer Inform. 2015; 14:141–8. doi: 10.4137/CIN.S34144 PMID: 26691437
9.
Caramuta S, Lee L, Ozata DM, Akcakaya P, Georgii-Hemming P, Xie H, et al. Role of microRNAs and
microRNA machinery in the pathogenesis of diffuse large B-cell lymphoma. Blood Cancer J. 2013; 3:
e152. doi: 10.1038/bcj.2013.49 PMID: 24121164
10.
Fang C, Zhu DX, Dong HJ, Zhou ZJ, Wang YH, Liu L, et al. Serum microRNAs are promising novel bio-
markers for diffuse large B cell lymphoma. Ann Hematol. 2012; 91(4):553–9. doi: 10.1007/s00277-011-
1350-9 PMID: 21987025
11.
Jorgensen LK, Poulsen MO, Laursen MB, Marques SC, Johnsen HE, Bogsted M, et al. MicroRNAs as
novel biomarkers in diffuse large B-cell lymphoma—a systematic review. Dan Med J. 2015; 62(5).
12.
Lawrie CH, Chi J, Taylor S, Tramonti D, Ballabio E, Palazzo S, et al. Expression of microRNAs in diffuse
large B cell lymphoma is associated with immunophenotype, survival and transformation from follicular
lymphoma. J Cell Mol Med. 2009; 13(7):1248–60. doi: 10.1111/j.1582-4934.2008.00628.x PMID:
19413891
13.
Lim EL, Trinh DL, Scott DW, Chu A, Krzywinski M, Zhao Y, et al. Comprehensive miRNA sequence
analysis reveals survival differences in diffuse large B-cell lymphoma patients. Genome Biol. 2015;
16:18. doi: 10.1186/s13059-014-0568-y PMID: 25723320
14.
Troppan K, Wenzl K, Deutsch A, Ling H, Neumeister P, Pichler M. MicroRNAs in diffuse large B-cell
lymphoma: implications for pathogenesis, diagnosis, prognosis and therapy. Anticancer Res. 2014;
34(2):557–64. PMID: 24510984
15.
Jasinski-Bergner S, Mandelboim O, Seliger B. The role of microRNAs in the control of innate immune
response in cancer. J Natl Cancer Inst. 2014; 106(10).
16.
Raisch J, Darfeuille-Michaud A, Nguyen HT. Role of microRNAs in the immune system, inflammation
and cancer. World J Gastroenterol. 2013; 19(20):2985–96. doi: 10.3748/wjg.v19.i20.2985 PMID:
23716978
17.
Suzuki H, Maruyama R, Yamamoto E, Kai M. DNA methylation and microRNA dysregulation in cancer.
Mol Oncol. 2012; 6(6):567–78. doi: 10.1016/j.molonc.2012.07.007 PMID: 22902148
18.
Macfarlane LA, Murphy PR. MicroRNA: Biogenesis, Function and Role in Cancer. Curr Genomics.
2010; 11(7):537–61. doi: 10.2174/138920210793175895 PMID: 21532838
19.
Mazan-Mamczarz K, Gartenhaus RB. Role of microRNA deregulation in the pathogenesis of diffuse
large B-cell lymphoma (DLBCL). Leuk Res. 2013; 37(11):1420–8. doi: 10.1016/j.leukres.2013.08.020
PMID: 24054860
20.
Di Lisio L, Sanchez-Beato M, Gomez-Lopez G, Rodriguez ME, Montes-Moreno S, Mollejo M, et al.
MicroRNA signatures in B-cell lymphomas. Blood Cancer J. 2012; 2(2):e57. doi: 10.1038/bcj.2012.1
PMID: 22829247
21.
Kozloski GA, Jiang X, Bhatt S, Ruiz J, Vega F, Shaknovich R, et al. miR-181a negatively regulates NF-
kappaB signaling and affects activated B-cell-like diffuse large B-cell lymphoma pathogenesis. Blood.
2016; 127(23):2856–66. doi: 10.1182/blood-2015-11-680462 PMID: 26941399
22.
Harries LW. MicroRNAs as Mediators of the Ageing Process. Genes (Basel). 2014; 5(3):656–70.
23.
Wang WT, Chen YQ. Circulating miRNAs in cancer: from detection to therapy. J Hematol Oncol. 2014;
7:86. doi: 10.1186/s13045-014-0086-0 PMID: 25476853
24.
Schwarzenbach H, Nishida N, Calin GA, Pantel K. Clinical relevance of circulating cell-free microRNAs
in cancer. Nat Rev Clin Oncol. 2014; 11(3):145–56. doi: 10.1038/nrclinonc.2014.5 PMID: 24492836
25.
Stankova M, Kubaczkova V, Sedlarikova L, Sevcikova S. Circulating microRNA as Biomarkers in
Hematological Malignancies. EXS. 2015; 106:123. doi: 10.1007/978-3-0348-0955-9_5 PMID:
26608201
26.
Ameling S, Kacprowski T, Chilukoti RK, Malsch C, Liebscher V, Suhre K, et al. Associations of circulat-
ing plasma microRNAs with age, body mass index and sex in a population-based study. BMC Med
Genomics. 2015; 8:61. doi: 10.1186/s12920-015-0136-7 PMID: 26462558
27.
Hatse S, Brouwers B, Dalmasso B, Laenen A, Kenis C, Schoffski P, et al. Circulating MicroRNAs as
easy-to-measure aging biomarkers in older breast cancer patients: correlation with chronological age
but not with fitness/frailty status. PLoS One. 2014; 9(10):e110644. doi: 10.1371/journal.pone.0110644
PMID: 25333486
28.
Noren Hooten N, Fitzpatrick M, Wood WH 3rd, De S, Ejiogu N, Zhang Y, et al. Age-related changes in
microRNA levels in serum. Aging (Albany NY). 2013; 5(10):725–40.
29.
Inada K, Okoshi Y, Cho Y, Saito H, Iijima T, Hori M, et al. Availability of Circulating MicroRNAs as a Bio-
marker for Early Diagnosis of Diffuse Large B-Cell Lymphoma. Open Journal of Blood Diseases. 2015;
5:48–58.
miRNA Predictions for Lymphomagenesis
PLOS ONE | DOI:10.1371/journal.pone.0170521
January 20, 2017
14 / 16


# Página 15

30.
Witwer KW. Circulating microRNA biomarker studies: pitfalls and potential solutions. Clin Chem. 2015;
61(1):56–63. doi: 10.1373/clinchem.2014.221341 PMID: 25391989
31.
Zheng RL, Jiang YJ, Wang X. Role of microRNAs on therapy resistance in Non-Hodgkin’s lymphoma.
Int J Clin Exp Med. 2014; 7(11):3818–32. PMID: 25550890
32.
Ramkumar C, Cui H, Kong Y, Jones SN, Gerstein RM, Zhang H. Smurf2 suppresses B-cell proliferation
and lymphomagenesis by mediating ubiquitination and degradation of YY1. Nat Commun. 2013;
4:2598. doi: 10.1038/ncomms3598 PMID: 24121673
33.
Ramkumar C, Kong Y, Cui H, Hao S, Jones SN, Gerstein RM, et al. Smurf2 regulates the senescence
response and suppresses tumorigenesis in mice. Cancer Res. 2012; 72(11):2714–9. doi: 10.1158/
0008-5472.CAN-11-3773 PMID: 22552287
34.
Li Y, Kowdley KV. Method for microRNA isolation from clinical serum samples. Anal Biochem. 2012;
431(1):69–75. doi: 10.1016/j.ab.2012.09.007 PMID: 22982505
35.
Bindea G, Galon J, Mlecnik B. CluePedia Cytoscape plugin: pathway insights using integrated experi-
mental and in silico data. Bioinformatics. 2013; 29(5):661–3. doi: 10.1093/bioinformatics/btt019 PMID:
23325622
36.
Ramkumar C, Kong Y, Trabucco SE, Gerstein RM, Zhang H. Smurf2 regulates hematopoietic stem cell
self-renewal and aging. Aging Cell. 2014; 13(3):478–86. doi: 10.1111/acel.12195 PMID: 24494704
37.
Beheshti A, Benzekry S, McDonald JT, Ma L, Peluso M, Hahnfeldt P, et al. Host age is a systemic regu-
lator of gene expression impacting cancer progression. Cancer Res. 2015; 75(6):1134–43. doi: 10.
1158/0008-5472.CAN-14-1053 PMID: 25732382
38.
Beheshti A, Wage J, McDonald JT, Lamont C, Peluso M, Hahnfeldt P, et al. Tumor-host signaling inter-
action reveals a systemic, age-dependent splenic immune influence on tumor development. Oncotar-
get. 2015; 6(34):35419–32. doi: 10.18632/oncotarget.6214 PMID: 26497558
39.
Hindson CM, Chevillet JR, Briggs HA, Gallichotte EN, Ruf IK, Hindson BJ, et al. Absolute quantification
by droplet digital PCR versus analog real-time PCR. Nat Methods. 2013; 10(10):1003–5. doi: 10.1038/
nmeth.2633 PMID: 23995387
40.
Doi H, Takahara T, Minamoto T, Matsuhashi S, Uchii K, Yamanaka H. Droplet digital polymerase chain
reaction (PCR) outperforms real-time PCR in the detection of environmental DNA from an invasive fish
species. Environ Sci Technol. 2015; 49(9):5601–8. doi: 10.1021/acs.est.5b00253 PMID: 25850372
41.
Tang W, Zhu J, Su S, Wu W, Liu Q, Su F, et al. MiR-27 as a prognostic marker for breast cancer pro-
gression and patient survival. PLoS One. 2012; 7(12):e51702. doi: 10.1371/journal.pone.0051702
PMID: 23240057
42.
Nie K, Zhang T, Allawi H, Gomez M, Liu Y, Chadburn A, et al. Epigenetic down-regulation of the tumor
suppressor gene PRDM1/Blimp-1 in diffuse large B cell lymphomas: a potential role of the microRNA
let-7. Am J Pathol. 2010; 177(3):1470–9. doi: 10.2353/ajpath.2010.091291 PMID: 20651244
43.
Trang P, Medina PP, Wiggins JF, Ruffino L, Kelnar K, Omotola M, et al. Regression of murine lung
tumors by the let-7 microRNA. Oncogene. 2010; 29(11):1580–7. doi: 10.1038/onc.2009.445 PMID:
19966857
44.
Sandhu SK, Croce CM, Garzon R. Micro-RNA Expression and Function in Lymphomas. Adv Hematol.
2011; 2011:347137. doi: 10.1155/2011/347137 PMID: 21461378
45.
Troppan K, Wenzl K, Pichler M, Pursche B, Schwarzenbacher D, Feichtinger J, et al. miR-199a and
miR-497 Are Associated with Better Overall Survival due to Increased Chemosensitivity in Diffuse
Large B-Cell Lymphoma Patients. Int J Mol Sci. 2015; 16(8):18077–95. doi: 10.3390/ijms160818077
PMID: 26251897
46.
Yuan WX, Gui YX, Na WN, Chao J, Yang X. Circulating microRNA-125b and microRNA-130a expres-
sion profiles predict chemoresistance to R-CHOP in diffuse large B-cell lymphoma patients. Oncol Lett.
2016; 11(1):423–32. doi: 10.3892/ol.2015.3866 PMID: 26870228
47.
Gururajan M, Chui R, Karuppannan AK, Ke J, Jennings CD, Bondada S. c-Jun N-terminal kinase (JNK)
is required for survival and proliferation of B-lymphoma cells. Blood. 2005; 106(4):1382–91. doi: 10.
1182/blood-2004-10-3819 PMID: 15890690
48.
Zhou K, Xu D, Cao Y, Wang J, Yang Y, Huang M. C-MYC aberrations as prognostic factors in diffuse
large B-cell lymphoma: a meta-analysis of epidemiological studies. PLoS One. 2014; 9(4):e95020. doi:
10.1371/journal.pone.0095020 PMID: 24740248
49.
Cai Q, Medeiros LJ, Xu X, Young KH. MYC-driven aggressive B-cell lymphomas: biology, entity, differ-
ential diagnosis and clinical management. Oncotarget. 2015; 6(36):38591–616. doi: 10.18632/
oncotarget.5774 PMID: 26416427
50.
Calin GA, Liu CG, Sevignani C, Ferracin M, Felli N, Dumitru CD, et al. MicroRNA profiling reveals dis-
tinct signatures in B cell chronic lymphocytic leukemias. Proc Natl Acad Sci U S A. 2004; 101(32):
11755–60. doi: 10.1073/pnas.0404432101 PMID: 15284443
miRNA Predictions for Lymphomagenesis
PLOS ONE | DOI:10.1371/journal.pone.0170521
January 20, 2017
15 / 16


# Página 16

51.
Michael MZ, OC SM, van Holst Pellekaan NG, Young GP, James RJ. Reduced accumulation of specific
microRNAs in colorectal neoplasia. Mol Cancer Res. 2003; 1(12):882–91. PMID: 14573789
52.
Metzler M, Wilda M, Busch K, Viehmann S, Borkhardt A. High expression of precursor microRNA-155/
BIC RNA in children with Burkitt lymphoma. Genes Chromosomes Cancer. 2004; 39(2):167–9. doi: 10.
1002/gcc.10316 PMID: 14695998
53.
Eis PS, Tam W, Sun L, Chadburn A, Li Z, Gomez MF, et al. Accumulation of miR-155 and BIC RNA in
human B cell lymphomas. Proc Natl Acad Sci U S A. 2005; 102(10):3627–32. doi: 10.1073/pnas.
0500613102 PMID: 15738415
54.
Cimmino A, Calin GA, Fabbri M, Iorio MV, Ferracin M, Shimizu M, et al. miR-15 and miR-16 induce apo-
ptosis by targeting BCL2. Proc Natl Acad Sci U S A. 2005; 102(39):13944–9. doi: 10.1073/pnas.
0506654102 PMID: 16166262
55.
Jardin F, Figeac M. MicroRNAs in lymphoma, from diagnosis to targeted therapy. Curr Opin Oncol.
2013; 25(5):480–6. PMID: 23852382
56.
Puvvada S, Kendrick S, Rimsza L. Molecular classification, pathway addiction, and therapeutic target-
ing in diffuse large B cell lymphoma. Cancer Genet. 2013; 206(7–8):257–65. doi: 10.1016/j.cancergen.
2013.07.003 PMID: 24080457
57.
Gallamini A, Borra A. Role of PET in lymphoma. Curr Treat Options Oncol. 2014; 15(2):248–61. doi: 10.
1007/s11864-014-0278-4 PMID: 24619427
58.
Due H, Svendsen P, Bodker JS, Schmitz A, Bogsted M, Johnsen HE, et al. miR-155 as a Biomarker in
B-Cell Malignancies. Biomed Res Int. 2016; 2016:9513037. doi: 10.1155/2016/9513037 PMID:
27294145
59.
Lawrie CH, Gal S, Dunlop HM, Pushkaran B, Liggins AP, Pulford K, et al. Detection of elevated levels of
tumour-associated microRNAs in serum of patients with diffuse large B-cell lymphoma. Br J Haematol.
2008; 141(5):672–5. doi: 10.1111/j.1365-2141.2008.07077.x PMID: 18318758
60.
Wang X, Cao L, Wang Y, Wang X, Liu N, You Y. Regulation of let-7 and its target oncogenes (Review).
Oncol Lett. 2012; 3(5):955–60. doi: 10.3892/ol.2012.609 PMID: 22783372
61.
Cubillos-Ruiz JR, Baird JR, Tesone AJ, Rutkowski MR, Scarlett UK, Camposeco-Jacobs AL, et al.
Reprogramming tumor-associated dendritic cells in vivo using miRNA mimetics triggers protective
immunity against ovarian cancer. Cancer Res. 2012; 72(7):1683–93. doi: 10.1158/0008-5472.CAN-11-
3160 PMID: 22307839
62.
Kasinski AL, Slack FJ. Epigenetics and genetics. MicroRNAs en route to the clinic: progress in validat-
ing and targeting microRNAs for cancer therapy. Nat Rev Cancer. 2011; 11(12):849–64. doi: 10.1038/
nrc3166 PMID: 22113163
63.
Makino K, Jinnin M, Hirano A, Yamane K, Eto M, Kusano T, et al. The downregulation of microRNA let-
7a contributes to the excessive expression of type I collagen in systemic and localized scleroderma. J
Immunol. 2013; 190(8):3905–15. doi: 10.4049/jimmunol.1200822 PMID: 23509348
64.
Christopher AF, Kaur RP, Kaur G, Kaur A, Gupta V, Bansal P. MicroRNA therapeutics: Discovering
novel targets and developing specific therapy. Perspect Clin Res. 2016; 7(2):68–74. doi: 10.4103/2229-
3485.179431 PMID: 27141472
65.
Kim DG, Kim KH, Seo YJ, Yang H, Marcusson EG, Son E, et al. Anti-miR delivery strategies to bypass
the blood-brain barrier in glioblastoma therapy. Oncotarget. 2016.
miRNA Predictions for Lymphomagenesis
PLOS ONE | DOI:10.1371/journal.pone.0170521
January 20, 2017
16 / 16
